scripts/install/SetupDefaultKvPersistence.php

```diff
@@ -47,6 +49,12 @@ public function __invoke($params)
             $persistenceManager->registerPersistence('default_kv', $newPersistenceConfig);
             $report->add(Report::createInfo('Setup new "default_kv" persistence.'));
         }
+        $schemaCollection = $persistenceManager->getSqlSchemas();
+        $kvdriver = $persistenceManager->getPersistenceById('default_kv')->getDriver();
+        if ($kvdriver instanceof SchemaProviderInterface) {
```

Comment:
As this is an install script I would recommend to use logger here to output those actions here or lack of them.